api/specs/api-v3.0.yaml

```
@@ -0,0 +1,1453 @@
+  /certificates/{uuid}/attachment:
+    parameters:
+      - name: uuid
+        in: path
+        description: Identifiant du certificat précédemment créé.
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+      - Attestations
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)